lib/pbench/server/database/alembic/versions/1a91bc68d6de_update_api_key.py

```diff
@@ -20,11 +20,14 @@ def upgrade() -> None:
     op.drop_constraint("api_keys_pkey", "api_keys", type_="primary")
     op.execute("ALTER TABLE api_keys ADD COLUMN id SERIAL PRIMARY KEY")
     op.add_column("api_keys", sa.Column("name", sa.String(length=128), nullable=True))
-    op.create_unique_constraint("api_keys_api_key_key", "api_keys", ["api_key"])
+    op.add_column("api_keys", sa.Column("key", sa.String(length=500), nullable=False))
+    op.create_unique_constraint(None, "api_keys", ["key"])
```

Comment:
> The use case of alteration of existing tables and constraints can be handled by schema migration tools such as [Alembic](https://alembic.sqlalchemy.org/). However, neither Alembic nor SQLAlchemy currently create names for constraint objects where the name is otherwise unspecified, leading to the case where being able to alter existing constraints means that one must reverse-engineer the naming system used by the relational database to auto-assign names, or that care must be taken to ensure that all constraints are named.

When I modified the version file earlier, I based the commands on what `psql` actually showed, and apparently this is exactly what the documentation means by "one must reverse-engineer the naming system" ... So maybe we should be specifying names here to avoid that in the future. I don't particularly like the `api_keys_key_key` pattern, but using the name and column suggests that these are global names, so maybe `api_keys_key_unique`??